install_scripts/templates/kubernetes-yml-generate.sh

```diff
@@ -297,23 +272,72 @@ spec:
     port: 8800
     nodePort: ${UI_BIND_PORT}
     protocol: TCP
-  - name: replicated-registry
-    port: 9874
-    nodePort: 9874
-    protocol: TCP
-  - name: replicated-iapi
-    port: 9877
-    nodePort: 9877
-    protocol: TCP
-  - name: replicated-snapshots
-    port: 9878
-    nodePort: 9878
-    protocol: TCP
-  - name: replicated-support
-    port: 9881
-    nodePort: 9881
+EOF
+else
+    cat <<EOF
+---
+apiVersion: v1
+kind: Service
+metadata:
+  name: replicated-node
```

Comment:
should it always be called `-node`? what about calling them both `-ui`?

Reply:
that's better